Add Substrate

danforbes opened the pull request:
https://github.com/substrate-developer-hub/awesome-substrate

A curated lists of awesome projects and resources related to the Substrate blockchain development framework. 

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

## Requirements for your pull request

:heavy_check_mark:  **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
:heavy_check_mark:  **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**

> https://github.com/sindresorhus/awesome/pull/1867
> https://github.com/sindresorhus/awesome/pull/1866
> https://github.com/sindresorhus/awesome/pull/1859

:heavy_check_mark:  You have read and understood the [instructions for creating a list](create-list.md).
:heavy_check_mark:  This pull request has a title in the format `Add Name of List`.
:heavy_check_mark:  Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.

> [Substrate](https://substrate.dev/) - Framework for writing scalable, upgradeable blockchains in Rust.

:heavy_check_mark:  Your entry should be added at the bottom of the appropriate category.
:heavy_check_mark:  The suggested Awesome list complies with the below requirements.

:heavy_check_mark: :heavy_check_mark: :heavy_check_mark: **Go to the top and read it again.**

aureooms reviewed (changes requested):
Are you sure you have read all the guidelines?

aureooms commented on readme.md:
```diff
@@ -568,6 +568,7 @@
 - [EOSIO](https://github.com/DanailMinchev/awesome-eosio#readme) - A decentralized operating system supporting industrial-scale apps.
 - [Corda](https://github.com/chainstack/awesome-corda#readme) - Open source blockchain platform designed for business.
 - [Waves](https://github.com/msmolyakov/awesome-waves#readme) - Open source blockchain platform and development toolset for Web 3.0 apps and decentralized solutions.
+- [Substrate](https://github.com/substrate-developer-hub/awesome-substrate#readme) - Framework for writing scalable, upgradeable blockchains in Rust; crafted with ❤️ by [Parity Technologies](https://github.com/paritytech).
```
I believe the part after the `;` (crafted with ❤️ by [Parity Technologies](https://github.com/paritytech)) is noise.

aureooms replied:
You have not run `awesome-lint` on the list.

danforbes replied:
Thank you for pointing out the the `awesome-lint` problem :pray: Personally I do not believe the link to Parity's GitHub organization is noisy because it adds context to the entry and the two links do not refer to the same GitHub organization. I'm happy to remove it if other people agree that it is noisy.

aureooms replied:
@danforbes It says on your GitHub profile that you are a *developer advocate* for Parity Technologies. Are you not in conflict of interest?

danforbes commented:
Converting to draft pending https://github.com/substrate-developer-hub/awesome-substrate/issues/22

danforbes commented:
The `awesome-lint` issues were fixed by @CurlyBracketEffect :tada:

sindresorhus commented:
The header graphic should either incorporate the "Awesome" text and replace the text header title, or the logo should be made much smaller and moved to the right side.

---

The "What is Substrate" should be at the top without a section header.

---

`-GitHub Topics: Awesome Lists` should be moved to the bottom of the list, under a `Related` section.

---

`Substrate` in `Substrate is a framework for building upgradable` should be linkified to its official website.

---

> The description starts with an uppercase character and ends with a period. - https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md

danforbes commented:
Thank you @CurlyBracketEffect for addressing the issues raised by @sindresorhus :pray: I went back and adjusted a few things (e.g. I removed the link to the GitHub topic as opposed to including it as an "orphan" in a dedicated section) so hopefully all is still well. Sindre, I am not entirely certain what your last remark (starting with an uppercase character and ending with a dot) is in reference to - is it about the title of this Pull Request or something else?

CurlyBracketEffect commented:
> Sindre, I am not entirely certain what your last remark (starting with an uppercase character and ending with a dot) is in reference to - is it about the title of this Pull Request or something else?

@danforbes There were a couple of descriptions that did not end in a period, but I fixed that in my PR. Should be all good now .

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1329740677605777409

danforbes commented:
Ah, how did I miss this?!? Thank you so much 🙏🏻
